tests/utils/conda_package_helper.py

```diff
@@ -41,58 +40,36 @@ class CondaPackageHelper:
     """Conda package helper permitting to get information about packages"""
 
     def __init__(self, container: TrackedContainer):
-        self.running_container: Container = CondaPackageHelper.start_container(
-            container
-        )
-        self.requested: dict[str, set[str]] | None = None
-        self.installed: dict[str, set[str]] | None = None
-        self.available: dict[str, set[str]] | None = None
-        self.comparison: list[dict[str, str]] = []
-
-    @staticmethod
-    def start_container(container: TrackedContainer) -> Container:
-        """Start the TrackedContainer and return an instance of a running container"""
+        self.container = container
         LOGGER.info(f"Starting container {container.image_name} ...")
-        return container.run_detached(
+        container.run_detached(
             tty=True,
             command=["bash", "-c", "sleep infinity"],
         )
 
-    @staticmethod
-    def _conda_export_command(from_history: bool) -> list[str]:
```

Comment:
In my opinion this adds complexity, but doesn't add too much value, so removed this method